crates/red_knot_python_semantic/resources/mdtest/shadowing/class.md

```diff
@@ -3,7 +3,9 @@
 ## Implicit error
 
 ```py
-class C: pass
+class C: ...
```

Comment:
I didn't realize Black preferred ellipsis over `pass`, even in non-stub files?

Reply:
It does not! This change was made manually by me in commit 3 of this PR: "[Manual cleanups following autoformatting](https://github.com/astral-sh/ruff/pull/13809/commits/adafb79b5c46647e3092153feaa9e14e66dcd6bf)".

My motivation for changing the `pass` to `...` is that if you use `...`, Black figures that it's a stub-like class definition, so you probably want stub-like formatting.

(It lets you put the whole class definition on multiple lines with `...`. If you use `pass`, it splits it over two lines.)